Source/Views/Cells/ImageAndTextCell.m

```diff
@@ -47,9 +47,7 @@ - (id)init
 }
 
 - (void)dealloc {
-	[image release];
 	image = nil;
-	[super dealloc];
 }
```

Comment:
Do we need this block? Or could it just go bye?

Reply:
It's safer to leave it until we convert the image to property